downstream/assemblies/platform/assembly-installing-controller-operator.adoc

```diff
@@ -28,6 +28,8 @@ When an instance of {ControllerName} is removed, the associated PVCs are not aut
 == Prerequisites
 
 * You have installed the {PlatformName} catalog in Operator Hub.
+* For Controller, a default 'StorageClass' must be configured on the cluster for the operator to dynamically create needed PVC's. This is not necessary if an external PostgreSQL database is configured.
```

Comment:
```suggestion
* For Controller, a default StorageClass must be configured on the cluster for the operator to dynamically create needed PVCs. This is not necessary if an external PostgreSQL database is configured.
```

Suggestion to remove apostrophe from `PVCs` `and StorageClass`.  Took a look at other RH guides and `StorageClass` does not have apostrophes. If this was directed by your SME, please disregard my suggestion.

Reply:
@lmalivert I made the corrections. I am still relying internally upon what i used to do. I appreciate the review!

Reply:
You're welcome!